static/index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
```

Comment:
Let's refer directly to our assets. If we need `webpack` at all, it will be to refer to a known bundle name for our own code.